markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+
+This tutorial will walk you through creating a working BigCommerce embedded checkout example deployed to [Heroku](https://heroku.com). Embedded checkout uses an iFrame to insert the Optimized One-Page Checkout from your BigCommerce store into another site. This is a good solution if you want to sell products from your BigCommerce store on another site without worrying about PCI compliance.
+
+You will need to use the [Channels, Sites and Routes](#) APIs to complete this tutorial. The Channels API allows you to create and manage sales channel listings across a merchant's channels. The Sites and Routes APIs let you set an external storefront domain and destination routes that are not BigCommerce URLs.
+
+This tutorial will cover the sequence of API calls necessary to create an embedded checkout. In the real world, your app would manage all of these API calls server side, after being prompted by a customer adding an item to cart on your external site's front end.
+
+<a href='#embedded-checkout_prerequisites' aria-hidden='true' class='block-anchor'  id='embedded-checkout_prerequisites'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 1: Prerequisites
+
+### Credentials
+
+Obtain a set of Store API credentials, which can be found in the Control Panel of your BigCommerce store. See [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication#authentication_getting-api-credentials) for details.
+
+These scopes are required for this guide:
+
+- **Carts: Modify**\
+  Enables creating carts and requesting secure redirect links to checkout
+- **Channel Settings: Modify**\
+  Enables creation of a channel that references the external storefront
+- **Sites & Routes: Modify**\
+  Enables creation of routes to be used when directing the shopper back to the site
+- **Products: Read Only**\
+  Enables reading information from the product catalog
+
+We recommend limiting scopes only to what you need in your script or app. Learn more about scopes [here](https://developer.bigcommerce.com/api-docs/getting-started/authentication).
+
+### Create React App
+
+This tutorial uses several npm packages to build an embeddable checkout. First we will spin up a basic [React](https://reactjs.org) application that we'll use as a starting point. Run the following in your shell:
+
+<!--
+title: "Create react app"
+subtitle: ""
+lineNumbers: true
+-->
+
+```shell
+npx create-react-app bc-embedded-checkout
+
+cd bc-embedded-checkout
+
+npm start
+```
+
+### BigCommerce Checkout SDK
+Install the BigCommerce Checkout SDK to invoke a method that can render the checkout in your app.
```

Comment:
For clarity, suggest changing this:

> Install the BigCommerce Checkout SDK to invoke a method that can render the checkout in your app.

To this:
Install the BigCommerce Checkout SDK, which includes a method for rendering the embeddable checkout in your app.